TODO.md

```diff
@@ -0,0 +1,16 @@
+- Upgrade @taquito
+- Clean `yarn` `resolutions`
+- Is this error new? In console of running iOS build:
+```bash
+  Possible Unhandled Promise Rejection (id: 0):
+  Error: [messaging/unknown] The operation couldn’t be completed. No APNS token specified before fetching FCM Token
```

Comment:
These errors are related to firebase-messaging, which is used for push notifications, which don't work on a simulators. Does this errors appear on a real device?